meta/src/lib/layout/header/header.tsx

```diff
@@ -0,0 +1,37 @@
+import { type FC, useContext, useEffect } from "react";
+import { ThemeContext } from "styled-components";
+
+import Logo from "./assets/ejb-black-logo-v2.png";
+import { NavLinks, SiteBrand, StyledNav, StyledNavLink } from "./header.styled";
+
+const Header: FC = () => {
+  const themeContext = useContext(ThemeContext);
+
+  useEffect(() => (
+    console.log("Current theme: ", themeContext)
```

Comment:
Wanted to stop logging the current theme on page navigation and limit it to initial load and theme changes.